Specs/.eslintrc.json

```diff
@@ -4,7 +4,8 @@
         "jasmine": true
     },
     "rules": {
-        "no-self-assign": "off"
+        "no-self-assign": "off",
+        "no-duplicate-imports":"off"
```

Comment:
I know that a lot of our specs have duplicate imports due to the fact that we migrated from es6 modules and replaced the requirejs-style imports via script. However, I think this is something we'd like to clean up. 

That being said, it's probably best to do that in a follow up PR and a single commit for merging sake.

(I don't think there's an automatic `--fix` option for this rule unfortunately, but I think there's plugin you could use locally to make the change easier.)

Reply:
Ok, I'll look into that next.